src/infi/clickhouse_orm/models.py

```diff
@@ -160,7 +161,19 @@ def to_tsv(self, insertable_only=False):
         data = self.__dict__
         fields = self._fields
         if insertable_only:
-            fields = [f for f in fields if f[1].is_insertable()]
+            fields = [f for f in fields if not f[1].readonly]
         return '\t'.join(field.to_db_string(data[name], quote=False) for name, field in fields)
 
+    def to_dict(self, insertable_only=False, field_names=None, timezone_in_use=pytz.utc):
```

Comment:
Shouldn't insertable_only use the same terminology as "readonly"? I would call the flag "include_readonly" which reverses its logic: passing True returns all fields, passing False returns only writable fields.